Cart-wishlist/wishlist.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="wishlist.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://kit.fontawesome.com/272d1ec438.js" crossorigin="anonymous"></script>
+
+</head>
+<body>
+    <div class="container">
+        <!-- Navigation bar -->
+        <nav>
+            <div class="nav-section">
+                <label class="royal-lebel">Royal</label>
+                <ul class="nav-home-shop-link">
+                    <a class="nav-link" href="/index.html">Home</a>
+                    <a  class="nav-link" href="/Product-list/product-list.html">Shop</a>
+                </ul>
+            </div>
+            <div class="nav-section">
+                <div class="outer-search-bar-div">
+                    <button class="btn"><img class="zoom" src="/Image/Zoom.png" height="38" width="32"></button>
+                    <input type="text" class="search-bar" placeholder="Search..">
+                </div>
+                <ul class="nav-icons">
+                    <i class='fas fa-user-alt' style='font-size:20px'>
+                        <a  class="nav-link icon-content" href="/Authentication/Login.html">Login </a>
+                    </i>
+                    <i class="fa fa-heart" style='font-size:20px'>
+                        <a class="nav-link icon-content" href="wishlist.html">Wishlist</a>
+                    </i>
+                    <i class="fa fa-shopping-cart" style='font-size:20px'>
+                        <a class="nav-link icon-content" href="cart.html">Cart</a>
+                    </i>
+                </ul>
+            </div>
+        </nav>
+        <!--  First card -->
+        <div class="outer-product-side-bar">
+            <div class="product-container">
+                <h2>My<span>Wishlist</span> </h2>
```

Comment:
What does span do here, any styling purpose?

Reply:
Here I have used span for inline elemnet.